api/nodes/views.py

```diff
@@ -212,7 +212,8 @@ class NodeList(JSONAPIBaseView, bulk_views.BulkUpdateJSONAPIView, bulk_views.Bul
 
     # overrides NodesFilterMixin
     def get_default_queryset(self):
-        return self.optimize_node_queryset(default_node_list_permission_queryset(user=self.request.user, model_cls=Node))
```

Comment:
`optimize_node_queryset` was found to perform poorly at scale in groups/guardian testing. removing here - groups/guardian more broadly removes this, and is only using it in a place where there are actual speed improvements.